blog/_posts/2017-02-23-scalafix-v0.3.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: blog
+post-type: blog
+by: Ólafur Páll Geirsson
+title: "Refactor with scalafix v0.3"
+---
+
+I am happy to announce the release of [scalafix v0.3][scalafix].
+This release leverages the new scala.meta semantic API to provide a re-designed `Rewrite` and `Patch` API for refactorings.
```

Comment:
Good catch! I totally agree, "leverage our market position/capital/..." is such corp speak